metricbeat/Makefile

```diff
@@ -90,6 +90,6 @@ run-module:
 
 .PHONY: test-module
 test-module: ## @testing Tests the given module. Needs $MODULE as param an run-module must be started first.
-test-module: python-env
+test-module: python-env update metricbeat.test
```

Comment:
The part I worry now is that `test-module` was thought of a quick way to run the tests again and again during development and now each time update and the build process is run. But I definitively see your point that in case the code is changed, the above is needed anyways. LGTM for now :-)

Reply:
Maybe a new target could be added later with the specific things needed by `test-module`, for example is it needed to update the docs to this quick tests on modules?

Reply:
Yes, that is a good idea. The other thing we are working on is converting more and more things of `update` which are in python to Golang, means the python-env is not needed anymore which makes things much quicker.